package.json

```diff
@@ -9,7 +9,7 @@
   "private": true,
   "scripts": {
     "test:fullfunctional": "yarn playwright install chromium && yarn playwright test --project=chromium --grep @smoke",
-    "test:functional-chromium": "yarn playwright install chromium && yarn playwright test --project=chromium",
+    "test:functional-chromium": "yarn playwright install chromium && yarn playwright test --project=chromium --grep-invert @accessibility",
```

Comment:
is there a reason to disable accessibility tests?

Reply:
Yes, we have a separate accessibility step in pipeline [here](https://github.com/hmcts/et-xui-e2e-tests/blob/master/package.json#L15) and disabling it to run within functional step

Reply:
Thanks Gokul, sorry for stupid question, but I think after this change, tests with @accessibility tag won't run in chrome? Since we have the same set of UI tests, do we really need  "test:functional-chromium" script then?
Also, I think this is still work in progress because I don't see any individual tests with accessibility tag?